src/main/utils/translators.ts

```diff
@@ -23,14 +24,28 @@ export class Translators {
         );
     }
 
-    public static toSeverity(clientSeverity: ClientSeverity): Severity {
+    public static severityCountToIssues(clientIssue: ISeverityCount, gocenter_security_url: string): Issue[] {
+        let issues: Issue[] = [];
+        if (clientIssue) {
+            for (let [key, value] of Object.entries(clientIssue)) {
```

Comment:
It is hard to understand what are 'key' and 'value'. Please give them meaningful names.

Reply:
key ---> issueLevel
value ---> issueCount